gstd/gstd_http.c

```diff
@@ -281,9 +303,9 @@ do_request (SoupServer * server, SoupMessage * msg, GHashTable * query,
     ret = do_put (server, msg, name, &output, path, session);
   } else if (msg->method == SOUP_METHOD_DELETE) {
     ret = do_delete (server, msg, name, &output, path, session);
-  } else if (msg->method == SOUP_METHOD_OPTIONS){
+  } else if (msg->method == SOUP_METHOD_OPTIONS) {
     ret = GSTD_EOK;
-  } 
+  }
```

Comment:
Are we considering the case where the request is none of theses?
else { default case }

Reply:
Yes, `ret` is initialized with a default message.